tests/fixtures/uv_project/pyproject.toml

```diff
@@ -0,0 +1,21 @@
+[project]
+name = "uv-project"
+version = "0.1.0"
+description = "Add your description here"
+readme = "README.md"
+requires-python = ">=3.12"
+dependencies = []
```

Comment:
Do you think it's worth declaring a dependency here to exercise the integration more fully?

If yes there's a locally vendored package you can use at https://github.com/nat-n/poethepoet/blob/9997e878d4398b406c18ea7b23a49a176ec4c1a5/tests/fixtures/poetry_plugin_project/pyproject.toml#L11